bc_obps/bc_obps/settings.py

```diff
@@ -172,7 +172,7 @@
 # Only enable sentry in production
 SENTRY_ENVIRONMENT = os.environ.get('SENTRY_ENVIRONMENT')
 SENTRY_TRACE_SAMPLE_RATE = os.environ.get('SENTRY_TRACE_SAMPLE_RATE')
-if SENTRY_ENVIRONMENT == 'prod' and DEBUG == 'False':
+if SENTRY_ENVIRONMENT == 'prod':
```

Comment:
I don't think I understand the problem/fix here. If we were never passing a DEBUG variable then DEBUG would be None which is still falsey in python right?

Reply:
We are strictly comparing the variable `DEBUG` with the string `"False"`, but since we never pass `DEBUG` in any of our environment variables, it will always be `None`. Therefore, it doesn't matter to keep this extra check here. We are already making sure that the `SENTRY_ENVIRONMENT` is set to `"prod"`, which should be sufficient.

Reply:
Gotcha, thanks!